src/manage_nsfs/health.js

```diff
@@ -66,6 +66,10 @@ const health_errors = {
         error_code: 'INVALID_CONFIG_DIR',
         error_message: 'Config directory is invalid',
     },
+    NOTIF_LOW_STORAGE: {
```

Comment:
@alphaprinz are we using this const somewhere?

Reply:
No, I've removed it.